instructions.md

```diff
@@ -69,6 +70,9 @@ For each graph, the event with the highest magnitude is labeled as the **mainsho
 
 Different clusters can be compared easily by selecting a time range in each of the two tabs. **NB**: The clustering takes quite a while, especially with large amounts of data.
 
+When using the clustering feature with FENCAT data, please note that earthquakes that occurred before the year 
+1678 cannot be processed. This mean that about 27 earthquakes are left out. The issue is caused by datetime formatting, which doesn't allow for dates that far in the past.
```

Comment:
This being a new line makes the text in the app have a larger gap than it is supposed to have.